### PR TITLE
Add test for force transfer.

### PR DESCRIPTION
**Pull request by nfurfaro**

### CODE REVIEW

**Comment by adlerjohn** [comment]
Until this is tested as part of CI, can you post explicit step-by-step instructions that can be copy-pasted (no playing around, literally copy-paste) to run the tests? Could be in the PR description for any PR that touches the stdlib, or maybe in the stdlib's README

**Comment by nfurfaro** [comment]
>can you post explicit step-by-step instructions that can be copy-pasted

Yes, will do that.

**Comment by nfurfaro** [comment]
@John Adler Would you be ok if I remove the `cargo test` command from `build.sh`?
Most of the time I'm passing flags to `cargo test`, and would prefer having `build.sh` do 1 thing only.
I found myself writing instructions for modifying `build.sh` as I'm doing that already, and thought its probably cleaner to just modify the script.

**Comment by adlerjohn** [comment]
Yes

File: build.sh [hunk 1]
```diff
@@ -7,7 +7,7 @@ do
 if [ -d "${f}" ];
 then
 echo "building test $f..."
-  forc build -o temp -p $f
+  /Users/furnic/dev/fuel/sway/target/debug/forc build -o temp -p $f
```
Comment (nfurfaro) [inline build.sh:10]: dang, fixing...

File: sway-lib-std/tests/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# Building & running the sway-lib-std tests
+
+As we currently don't have a CI job which runs these automatically, make sure tests are all passing before requesting a PR review or merging to master.
+
```
Comment (adlerjohn) [inline sway-lib-std/tests/README.md:4]:
```suggestion
```

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -146,6 +146,35 @@ jobs:
           command: run
           args: --release --bin test
 
+  # TODO: Remove this upon merging std tests with the rest of the E2E tests.
+  cargo-test-lib-std:
```
[the remark this answers is not in the document]
Comment (nfurfaro) [reply .github/workflows/ci.yml:150]: would that be the `build-publish-master-image` & `build-publish-release-image` jobs?

[the remark this answers is not in the document]
Comment (adlerjohn) [reply .github/workflows/ci.yml:150]: Whichever are the ones that have a long list of `needs` and that already need `build-sway-lib-std`

**Review by adlerjohn — APPROVED** [review]

**Pull request merged**